setup.py

```diff
@@ -15,6 +15,7 @@
           'backoff==1.8.0',
           'google-ads==15.0.0',
           'protobuf==3.17.3',
+          'grpcio-status==1.44.0',
```

Comment:
We can add some context here so it doesn't accidentally get removed, since it's not being imported anywhere but is a subtle requirement.

```suggestion
          # Necessary to handle gRPC exceptions properly, documented
          # in an issue here: https://github.com/googleapis/python-api-core/issues/301
          'grpcio-status==1.44.0',
```